### PR TITLE
🎉 Bring in the mcp ui extension from https://github.com/feedhenry/mcp-openshift-ui-extension

### PR DESCRIPTION
**Pull request by david-martin**
@maleck13 After we merge this, we can  look at a unified installation/setup for local dev

### CODE REVIEW

File: ui/README.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
...
+## Local Installation
+
+Start openshift, configure it for MCP Extension development, and install the mobile apiserver
+
+```
+./install.sh
+```
+
+The MCP Extension should now be visible in the OpenShift Web Console after the 'origin' container has restarted.
+Visit https://127.0.0.1:8443 to see the Console.
```
Comment (maleck13) [inline ui/README.md:18]: Would it be worth saying. Use oc to create some mobile apps and you should see them in the ui? To show that integration point?

File: ui/install.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+set -x
+set -e
+
+# Mobile API Server repo location
+MOBILE_API_SERVER_REPO=github.com/feedhenry/mobile-apiserver
```
Comment (maleck13) [inline ui/install.sh]: this is no longer correct. It should be ```github.com/feedhenry/mobile-control-panel/server```

File: ui/install.sh [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+MOBILE_API_SERVER_DIR=$GOPATH/src/$MOBILE_API_SERVER_REPO
+MOBILE_API_SERVER_INSTALL_SCRIPT=$MOBILE_API_SERVER_DIR/hack/install-apiserver/openshift/install.sh
+
+# Pull down the mobile-api server repo if not already cloned
+[[ -d $MOBILE_API_SERVER_DIR ]] || go get -u $MOBILE_API_SERVER_REPO
```
Comment (maleck13) [inline ui/install.sh]: No need to do this anymore as it will be part of this repo

File: ui/install.sh [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+# Start OpenShift with the current directory as the config dir
+# Using the current directory as the config dir is important for extension development
+# This allows changing of extension files and having them already mounted in the origin container
+oc cluster up --service-catalog --host-config-dir=${PWD} --version='v3.6.0-rc.0'
```
Comment (maleck13) [inline ui/install.sh]: something we can consider here is ``` --use-existing-config=true``` to stop it nuking the config each time?

File: ui/package.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  "name": "mcp-openshift-ui-extension",
+  "version": "1.0.0",
+  "description": "",
+  "main": "mcp.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/feedhenry/mcp-openshift-ui-extension.git"
```
Comment (maleck13) [inline ui/package.json]: prob needs to be updated now

**Review by maleck13 — CHANGES_REQUESTED** [review]
few changes to the scripts required

File: ui/install.sh (hunk 2 above)
Comment (david-martin) [reply ui/install.sh]: given that the mobile-apiserver code is in the same repo as this script now, i think a relative path woudl be better in cases where someone has cloned to a different dir

**Comment by david-martin** [comment]
@maleck13 Updated in 37c9a1c based on feedback

**Review by maleck13 — APPROVED** [review]

**Pull request merged**